### PR TITLE
Add support for DNF package manager, add required build dependencies

### PR DESCRIPTION
**Pull request by anadahz**
- Fedora 22 introduces the next upcoming major version of Yum DNF, this commit adds support for dnf package manager which is the default for Fedora 22
 - Add required package dependencies: GeoIP-devel, python-devel libdnet-devel gcc-c++

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -160,74 +167,27 @@ install_obfs4proxy() {
...
-}
-
-
-setup_backports() {
-  echo "deb http://ftp.de.debian.org/debian/ ${distro_codename}-backports main" > /etc/apt/sources.list.d/stable.list
-  $sh_c "gpg --keyserver pgpkeys.mit.edu --recv-key A1BD8E9D78F7FE5C3E65D8AF8B48AD6246925553"
-  $sh_c "gpg -a --export A1BD8E9D78F7FE5C3E65D8AF8B48AD6246925553 | apt-key add -"
-  $sh_c "apt-get update"
-}
-
```
Comment (hellais) [inline scripts/install.sh]: Why did you remove the installation of meek and setup of backports?

File: scripts/install.sh [hunk 2]
```diff
@@ -160,74 +167,27 @@ install_obfs4proxy() {
...
-install_go() {
```
Comment (hellais) [inline scripts/install.sh]: Why was this removed?

**Comment by hellais** [comment]
The commit message is a lie. You claim to only add support for Fedora, while you are also remove a lot of other things.

**Comment by anadahz** [comment]
Yes, testing OONI PR review eyes, good catch!
I guess it's safe to assume that this PR can be closed.

**Comment by hellais** [comment]
FYI in the future a good option is also to force push to the branch that has an open PR instead of re-opening another one.
